cachi2/core/package_managers/bundler/parser.py

```diff
@@ -120,6 +120,7 @@ class GitDependency(_GemMetadata):
     """
 
     url: AcceptedUrl
+    branch: Optional[str]
```

Comment:
```diff
-    branch: Optional[str]
+    branch: Optional[str] = None
```
It is (hopefully) a corner-case, I suggest not polluting everything just because of it.